app/src/components/ClinicSelectionList.tsx

```diff
@@ -0,0 +1,107 @@
+import { Trans } from 'next-i18next'
```

Comment:
Notes for reviewers: This component is broken out from `/choose-clinic` primarily so that we can call it as a [dynamic import](https://nextjs.org/docs/advanced-features/dynamic-import) and avoid hydration issues.